docs/pages/attributesJSON.rst

```diff
@@ -46,9 +46,125 @@ Below are the supported JSON tags for Physics Manager Attributes templates, and
 	- list of strings
 	- A list of locations to query for supported object files that should be available to be loaded into the world.
```

Comment:
Maybe point to another page referring to what kind of object files are supported.